### ISSUE TITLE
React Native: Cannot access "Server URL" field with iOS VoiceOver

### ISSUE THREAD

**Issue by esethna** [mattermost/mattermost-server#11955]
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

----

**Notes**: [Jira ticket](https://mattermost.atlassian.net/browse/MM-17649)

Users with VoiceOver activated cannot login on MM iOS app,

Repro: 
1. Log out of MM iOS app
2. Turn on iOS VoiceOver from Settings > General > Accessibility > VoiceOver
3. Return to MM and attempt to swipe right through the options on the Enter Server URL screen

Observed:
VoiceOver cannot access the server field and just repeats "Enter Server URL"

Expected: VoiceOver can scan through all the parts of this screen. ie "Enter Server URL" text, server URL type field, "Connect" button

Note: this probably also affects the login screen

**Comment by esethna** [comment]
This is client side, it would be on the mobile repo: https://github.com/mattermost/mattermost-mobile

**Comment by allenlai18** [comment]
i will look into this

**Comment by RyanCommits** [comment]
Looking into this

**Comment by migbot** [comment]
Thanks @RyanCommits!

**Comment by RyanCommits** [comment]
PR opened on the mobile repo https://github.com/mattermost/mattermost-mobile/pull/3855

**Comment by hanzei** [comment]
Fixed with https://github.com/mattermost/mattermost-mobile/pull/3855 within one day :tada: Thanks @RyanCommits
